docs/java/how-to-use-signals-in-java.md

```diff
@@ -0,0 +1,46 @@
+---
+id: how-to-use-signals-in-java
+title: How to use Signals in Java
+sidebar_label: Signals
+description: To use Signals in Java, initiate the Signal method with `@SignalMethod` annotation in the Workflow interface and call the Signal method either directly from the Client or from within another Workflow.
```

Comment:
To shorten the description and to parallel how-to-use-queries-in-java, how 'bout the following?
```suggestion
description: Initiate the Signal method with `@SignalMethod` annotation in the Workflow interface and call the Signal method either directly from the Client or from within another Workflow.
```